src/utils/event-manager/wheel-input.js

```diff
@@ -64,6 +64,7 @@ export default class WheelInput {
     if (!this.options.enable) {
       return;
     }
+    event.preventDefault();
```

Comment:
Calling `event.preventDefault` from the event handler for `wheel` event does not stop the default of the trackpad scroll gesture. There's a timing issue related to how event manager emits events.